robot:
  name: "nissan_leaf"
  type: "ackermann"
  kinematic_parameters:
    wheelbase: 2.7
    front_track: 1.5
    rear_track: 1.45
  wheelparameters:
    wheel_radius: 0.33
  